### ISSUE TITLE
Resources\Text\en\locations_en.txt

### ISSUE THREAD

**Issue by qhq** [Admiral-Fish/RaidFinder#59]
Resources\Text\en\locations_en.txt
I can only find the location name here now. If the location can be more detailed, the nest option can be better adapted to other versions. For example:
Rolling Fields 1
Rolling Fields 2
Rolling Fields 3
Rolling Fields 4
Rolling Fields 5
![snipaste20200310_130017](https://user-images.githubusercontent.com/3319385/76281578-6d9dd000-62d0-11ea-9e80-fdd1eedcf58a.png)
![snipaste20200310_130701](https://user-images.githubusercontent.com/3319385/76281581-6f679380-62d0-11ea-8a8d-d5bc4f2b04a9.jpg)

**Comment by Admiral-Fish** [comment]
I'm not sure I understand. What are you asking?

**Comment by qhq** [comment]
i mean the name of the dens is only seventeen in Resources\Text\en\locations_en.txt

> Axew's Eye
Bridge Field
Dappled Grove
Dusty Bowl
East Lake Axewell
Giant's Cap
Giant's Mirror
Giant's Seat
Hammerlocke Hills
Lake of Outrage
Motostoke Riverbank
North Lake Miloch
Rolling Fields
South Lake Miloch
Stony Wilderness
Watchtower Ruins
West Lake Axewell

It would be better if you could provide more details, For example:

> Rolling Fields 1
Rolling Fields 2
Rolling Fields 3
Rolling Fields 4
Rolling Fields 5

Because there's a way of naming A to Z
![下载](https://user-images.githubusercontent.com/3319385/76283274-2fef7600-62d5-11ea-9441-8b7433f8dd89.png)

**Comment by Admiral-Fish** [comment]
The locations get numbers added to them. I just didn't want a text file with 100 entries when I only need 17.

**Comment by qhq** [comment]
If I want to distinguish Rolling Fields 1 and Rolling Fields 2, I have to look at the ID number.
I can't know Rolling Fields A and Rolling Fields B by name.
I must remember that the Charizard is in the num 98.
What I found on other websites may be Lake of Outrage A.
Of course, this is not a big problem, but a personal suggestion.:)
Because I use the webpage to search Pokemon is show the name rather than the number.
![微信图片_20200310140514](https://user-images.githubusercontent.com/3319385/76284407-303d4080-62d8-11ea-819d-21c4225c5c44.png)

**Comment by Admiral-Fish** [comment]
Is there even a standardized naming for the different dens?

**Comment by qhq** [comment]
You can refer to this
> 
1 | Rolling Fields 1 | 煦丽草原 D
2 | Rolling Fields 2 | 煦丽草原 C
3 | Rolling Fields 3 | 煦丽草原 B
4 | Rolling Fields 4 | 煦丽草原 H
5 | Rolling Fields 5 | 煦丽草原 G
6 | Rolling Fields 6 | 煦丽草原 I
7 | Rolling Fields 7 | 煦丽草原 E
8 | Rolling Fields 8 | 煦丽草原 A
9 | Rolling Fields 9 | 煦丽草原 F
10 | Dappled Grove 1 | 沐光森林 E
11 | Dappled Grove 2 | 沐光森林 A
12 | Dappled Grove 3 | 沐光森林 B
13 | Dappled Grove 4 | 沐光森林 D
14 | Dappled Grove 5 | 沐光森林 C
15 | Watchtower Ruins 1 | 瞭望塔旧址 C
16 | Watchtower Ruins 3 | 瞭望塔旧址 A
17 | Watchtower Ruins 2 | 瞭望塔旧址 B
18 | East Lake Axewell 1 | 牙牙湖东岸 E
19 | East Lake Axewell 2 | 牙牙湖东岸 B
20 | East Lake Axewell 3 | 牙牙湖东岸 C
21 | East Lake Axewell 4 | 牙牙湖东岸 D
22 | West Lake Axewell 1 | 牙牙湖西岸 D
23 | West Lake Axewell 2 | 牙牙湖西岸 B
24 | West Lake Axewell 3 | 牙牙湖西岸 E
25 | West Lake Axewell 4 | 牙牙湖西岸 F
26 | West Lake Axewell 5 | 牙牙湖西岸 C
27 | West Lake Axewell 6 | 牙牙湖西岸 A
28 | Axew\'s Eye 1 | 牙牙湖之眼 A
29 | South Lake Miloch 1 | 美纳斯湖南岸 C
30 | South Lake Miloch 2 | 美纳斯湖南岸 D
31 | South Lake Miloch 3 | 美纳斯湖南岸 E
32 | South Lake Miloch 4 | 美纳斯湖南岸 B
33 | South Lake Miloch 5 | 美纳斯湖南岸 A
34 | Giant\'s Seat 1 | 巨人凳岩 D
35 | Giant\'s Seat 2 | 巨人凳岩 C
36 | Giant\'s Seat 3 | 巨人凳岩 B
37 | Giant\'s Seat 4 | 巨人凳岩 E
38 | Giant\'s Seat 5 | 巨人凳岩 A
39 | North Lake Miloch 1 | 美纳斯湖北岸 C
40 | North Lake Miloch 2 | 美纳斯湖北岸 F
41 | North Lake Miloch 3 | 美纳斯湖北岸 B
42 | North Lake Miloch 4 | 美纳斯湖北岸 E
43 | North Lake Miloch 5 | 美纳斯湖北岸 A
44 | East Lake Axewell 5 | 牙牙湖东岸 A
45 | North Lake Miloch 6 | 美纳斯湖北岸 D
46 | Motostoke Riverbank 1 | 机擎河岸 B
47 | Motostoke Riverbank 2 | 机擎河岸 A
48 | Motostoke Riverbank 3 | 机擎河岸 C
49 | Motostoke Riverbank 4 | 机擎河岸 D
50 | Bridge Field 1 | 桥间空地 E
51 | Bridge Field 2 | 桥间空地 H
52 | Bridge Field 3 | 桥间空地 F
53 | Bridge Field 4 | 桥间空地 I
54 | Bridge Field 5 | 桥间空地 B
55 | Bridge Field 6 | 桥间空地 D
56 | Bridge Field 7 | 桥间空地 A
57 | Bridge Field 8 | 桥间空地 C
58 | Bridge Field 9 | 桥间空地 G
59 | Stony Wilderness 1 | 巨石原野 J
60 | Stony Wilderness 2 | 巨石原野 C
61 | Stony Wilderness 3 | 巨石原野 L
62 | Stony Wilderness 4 | 巨石原野 A
63 | Stony Wilderness 5 | 巨石原野 B
64 | Stony Wilderness 6 | 巨石原野 D
65 | Stony Wilderness 7 | 巨石原野 F
66 | Stony Wilderness 8 | 巨石原野 K
67 | Stony Wilderness 9 | 巨石原野 H
68 | Stony Wilderness 10 | 巨石原野 I
69 | Stony Wilderness 11 | 巨石原野 G
70 | Stony Wilderness 12 | 巨石原野 E
71 | Dusty Bowl 1 | 沙尘洼地 G
72 | Dusty Bowl 2 | 沙尘洼地 C
73 | Dusty Bowl 3 | 沙尘洼地 A
74 | Dusty Bowl 4 | 沙尘洼地 B
75 | Dusty Bowl 5 | 沙尘洼地 D
76 | Dusty Bowl 6 | 沙尘洼地 I
77 | Dusty Bowl 7 | 沙尘洼地 E
78 | Dusty Bowl 8 | 沙尘洼地 F
79 | Giant\'s Mirror 1 | 巨人镜池 E
80 | Dusty Bowl 9 | 沙尘洼地 H
81 | Giant\'s Mirror 2 | 巨人镜池 D
82 | Giant\'s Mirror 3 | 巨人镜池 B
83 | Giant\'s Mirror 4 | 巨人镜池 A
84 | Giant\'s Mirror 5 | 巨人镜池 C
85 | Hammerlocke Hills 1 | 拳关丘陵 F
86 | Hammerlocke Hills 2 | 拳关丘陵 E
87 | Hammerlocke Hills 3 | 拳关丘陵 B
88 | Hammerlocke Hills 4 | 拳关丘陵 D
89 | Hammerlocke Hills 5 | 拳关丘陵 A
90 | Hammerlocke Hills 6 | 拳关丘陵 C
91 | Hammerlocke Hills 7 | 拳关丘陵 G
92 | Giant\'s Cap 1 | 巨人帽岩 E
93 | Giant\'s Cap 2 | 巨人帽岩 C
94 | Giant\'s Cap 3 | 巨人帽岩 D
95 | Giant\'s Cap 4 | 巨人帽岩 A
96 | Giant\'s Cap 5 | 巨人帽岩 B
97 | Lake of Outrage 1 | 逆鳞湖 D
98 | Lake of Outrage 2 | 逆鳞湖 A
99 | Lake of Outrage 3 | 逆鳞湖 C
100 | Lake of Outrage 4 | 逆鳞湖 B

**Comment by Admiral-Fish** [comment]
Yeah but is this like official Pokemon conventions? I really want to avoid site specific naming conventions unless it's considered an internet standard.

**Comment by qhq** [comment]
No official certification. It's more conventional. For example, the current 1-100 sorting is just a habit accepted by most people. A picture I sent earlier, a to Z naming method, is also a more popular naming habit on the Internet.
:D
[https://duckduckgo.com/?q=Pokemon+sword%2Fshield+raid+location+map&atb=v210-7a_&iax=images&ia=images](url)
